src/mongoose_client_api_rooms_users.erl

```diff
@@ -51,10 +51,10 @@ from_json(Req, #{user := User,
                                               UserToInvite, <<"member">>),
     {true, Req2, State};
 from_json(Req, State) ->
-    mongoose_client_api_rooms:forbidden_request(Req, State).
+    mongoose_client_api:forbidden_request(Req, State).
 
 delete_resource(Req, #{role_in_room := none} = State) ->
-    mongoose_client_api_rooms:forbidden_request(Req, State);
+    mongoose_client_api:forbidden_request(Req, State);
```

Comment:
404?

Reply:
It was 403 here before already :) If you want I can change to 404.

Reply:
PR is about "improve", not "refactor", so I think it's worth including in the same branch. :)